drivers/wifi/eswifi/eswifi_bus_spi.c

```diff
@@ -22,13 +22,21 @@ K_KERNEL_STACK_MEMBER(eswifi_spi_poll_stack, ESWIFI_SPI_THREAD_STACK_SIZE);
 
 #define SPI_READ_CHUNK_SIZE 32
 
+struct eswifi_spi_config {
+	struct gpio_dt_spec csn;
```

Comment:
this one is not initialized?

Reply:
was going to send a follow up PR to remove that after this got merged.